app/controllers/api/v1/pages_controller.rb

```diff
@@ -12,6 +13,35 @@ def show
         @page = Page.find params[:id]
         render json: @page
       end
+
+      def create
+        @page = Page.new permitted_params
+        result = @page.save
+        render json: @page, status: (result ? :ok : :unprocessable_entity)
+      end
+
+      def update
+        @page = Page.find(params[:id])
+        result = @page.update permitted_params
+        render json: @page, status: (result ? :ok : :unprocessable_entity)
+      end
+
+      def destroy
```

Comment:
It's a little concerning, but as far as I can tell, this is only the second time we've added a `destroy` end-point (the first is [in the `feature_flags` API concern](https://github.com/forem/forem/blob/main/app/controllers/concerns/api/feature_flags_controller.rb#L15-L18). I think this is more because our API has been growing slowly over time than any deliberate omission, but I could be wrong, so thought I should call it out?

Reply:
I am not fully certain on my answer, but my hunch is it's not a deliberate omission.

Reply:
+1 to what Mac said.